Epsilon.Abstractions/LearningDomainCriteria.cs

```diff
@@ -0,0 +1,6 @@
+﻿namespace Epsilon.Abstractions;
+
+public record LearningDomainCriteria(
+    int Id,
+    double? MasteryPoints
```

Comment:
Should probably be in outcome result

Reply:
When items are not graded, there is not a result generated. This is thus needed for the KPI Matrix

Reply:
Valid reason